user/caribou/module/src/DSO9254AEventConverter.cc

```diff
@@ -0,0 +1,377 @@
+#include "CaribouEvent2StdEventConverter.hh"
+#include "utils/log.hpp"
+#include "CLICTDFrameDecoder.hpp"
+#include "TFile.h"
+#include "TH1D.h"
+#include "time.h"
+
+using namespace eudaq;
+
+namespace{
+  auto dummy0 = eudaq::Factory<eudaq::StdEventConverter>::
+  Register<DSO9254AEvent2StdEventConverter>(DSO9254AEvent2StdEventConverter::m_id_factory);
+}
+
+uint64_t timeConverter( std::string date, std::string time );
+
+bool DSO9254AEvent2StdEventConverter::Converting(eudaq::EventSPC d1, eudaq::StandardEventSP d2, eudaq::ConfigurationSPC conf) const{
+  
+
+  auto ev = std::dynamic_pointer_cast<const eudaq::RawEvent>(d1);
+  // No event
+  if(!ev) {
+    return false;
+  }
+
+
+  // load parameters from config file
+  double pedStartTime = conf->Get("pedStartTime", 0 ); // integration windows in [ns]
+  double pedEndTime   = conf->Get("pedEndTime"  , 0 );
+  double ampStartTime = conf->Get("ampStartTime", 0 );
+  double ampEndTime   = conf->Get("ampEndTime"  , 0 );
+  bool generateRoot   = conf->Get("generateRoot", 0 ); 
+  std::cout << "Loaded parameters from configuration file." << std::endl
```

Comment:
Please do not use `std::cout` because then there is no way of controlling the verbosity level. This seems to be something appropriate for `EUDAQ_DEBUG`?